test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -227,19 +242,40 @@ test.requestHooks(requestLogger, mockEnrollAuthenticatorPassword)('select passwo
   await t.expect(enrollPasswordPage.passwordFieldExists()).eql(true);
   await t.expect(enrollPasswordPage.confirmPasswordFieldExists()).eql(true);
 
-  await t.expect(requestLogger.count(() => true)).eql(3);
-  const req1 = requestLogger.requests[0].request;
-  await t.expect(req1.url).eql('http://localhost:3000/idp/idx/introspect');
-
-  const req2 = requestLogger.requests[1].request;
-  await t.expect(req2.url).eql('http://localhost:3000/idp/idx/credential/enroll');
-  await t.expect(req2.method).eql('post');
-  await t.expect(req2.body).eql('{"authenticator":{"id":"autwa6eD9o02iBbtv0g3"},"stateHandle":"02CqFbzJ_zMGCqXut-1CNXfafiTkh9wGlbFqi9Xupt"}');
-
-  const req3 = requestLogger.requests[2].request;
-  await t.expect(req3.url).eql('http://localhost:3000/idp/idx/credential/enroll');
-  await t.expect(req3.method).eql('post');
-  await t.expect(req3.body).eql('{"authenticator":{"id":"autwa6eD9o02iBbtv0g3"},"stateHandle":"02CqFbzJ_zMGCqXut-1CNXfafiTkh9wGlbFqi9Xupt"}');
+  if (process.env.OKTA_SIW_V3) {
```

Comment:
do we really need to change the test spec? this does not seems like a parity test

Reply:
This is what I discussed in the thread yesterday. v3 makes an extra request that v2 does not make when switching authenticators.